spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'should be valid' do
+    user = User.create(username: '',
+                       email: '',
+                       password: '')
+    expect(user).to_not be_valid
+  end
+
+  it 'should create a new user' do
+    user = User.create(username: 'user 1',
+                       email: 'useremail@email.com',
+                       password: 'password')
+    expect(user.username).to eql('user 1')
```

Comment:
[OPTIONAL]
Should always have positive and negative tests to ensure it works the way you expect it. Eg. What do you expect the output to be when it's not valid or when valid etc.
Consider making all your tests more extensive, not just this one. For this simple application, it may not catch a possible crippling bug, but in bigger applications, it definitely can. Consider this for all projects moving forward.